test/run

```diff
@@ -27,6 +26,9 @@ case $TEST_SCENARIO in
         test/image-prepare --verbose --overlay $TEST_OS
         test/common/run-tests --jobs ${TEST_JOBS:-1} --test-dir test/verify
         ;;
+    mobile)
+        echo "Deprecated test scenario: $TEST_SCENARIO"
```

Comment:
may as well `exit 1` here.  Nobody has this status set as required.

Reply:
Yeah, this is just here to make the transition slightly more correct. If we remove fedora-35/mobile from testmap immediately when (or before) merging this, we probably don't need to be that correct.